libs/ledger-live-common/src/bot/index.ts

```diff
@@ -76,7 +76,7 @@ export async function bot({ currency, family, mutation }: Arg = {}) {
         if (mutation) {
           spec = {
             ...spec,
-            mutations: spec.mutation.filter((m) =>
+            mutations: spec.mutations.filter((m) =>
```

Comment:
wow, is this a bug? separate PR maybe?

Reply:
Yep, spotted and fixed by @emmanuelm41 :slightly_smiling_face: 
Here's a dedicated PR: https://github.com/LedgerHQ/ledger-live/pull/168
@gre no need to remove the fix from that PR right?